src/main/java/com/sumindaa/demo/employeeservice/model/Employee.java

```diff
@@ -1,5 +1,11 @@
 package com.sumindaa.demo.employeeservice.model;
 
-public class Employee {
+import java.util.Date;
 
+public class Employee {
+int id;
```

Comment:
Make these private and create getter setters

Reply:
done